### PR TITLE
T011 - Querying online API webservices

### PR DESCRIPTION
**Pull request by jaimergp**
# Details

* Talktorial ID: T011
* Title: Querying online API webservices
* Original authors: Jaime Rodríguez-Guerra
* Reviewer(s): David Schaller
* Date of review:

# Content review

* Potential labels or categories (e.g. machine learning, small molecules, online APIs): XXXXXX
* One line summary: XXXXXXXXXX
* [x] The table of contents reflects the talktorial story-line; order of #, ##, ### headers is correct
* [x] URLs are linked with meaningful words, instead of pasting the URL directly or linking words like `here`.
* [ ] I have spell-checked the notebook
* [x] Images have enough resolution to be rendered with quality, without being _too_ heavy.
* [ ] All figures have a description
* [x] Markdown cell content is still in-line with code cell output (whenever results are discussed)
* [x] I have checked that cell outputs are not incredibly long (this applies also to `DataFrames`)
* [x] Formatting looks correctly on the Sphinx render (bold, italics, figure placing)

# Code review

* Time it took to execute (approx.):
* [ ] Variable and function names follow snake case rules (e.g. `a_variable_name` vs `aVariableName`)
* [ ] Spacing follows PEP8 (run Black on the code cells if needed)
* [ ] Code line are under 99 characters each (run `black -l 99`)
* [ ] Comments are useful and well placed
* [ ] There are no unpythonic idioms like `for i in range(len(list))` (see slides)
* [ ] All 3rd party dependencies are listed at the top of the notebook
* [ ] I have marked all code cell with output referenced in markdown cells with the label `# TODO: CI`
* [ ] I have identified potential candidates for a code refactor / useful functions
* [ ] All `import ...` lines are at the top (practice part) cell, ordered by standard library / 3rd party packages / our own (`teachopencadd.*`)
* [ ] I have update the relative paths to absolute paths.
* List here unfamiliar libraries you find in the imports and their intended use:

### CODE REVIEW

**Comment by jaimergp** [comment]
@schallerdavid This is ready for review! 

@yonghui-cc, this can help you understand the basics of web APIs!

File: teachopencadd/talktorials/T011_query_online_api_webservices/talktorial.ipynb [hunk 1]
```diff
@@ -4,2811 +4,1348 @@
    "cell_type": "markdown",
```
Comment (schallerdavid) [inline teachopencadd/talktorials/T011_query_online_api_webservices/talktorial.ipynb:4]: <p>typo: parameter<s>e</s>s</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/volkamerlab/teachopencadd/pull/84/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='6'/>

Comment (schallerdavid) [inline teachopencadd/talktorials/T011_query_online_api_webservices/talktorial.ipynb:4]: <p>missing word: Ok, now let's <strong>get</strong> to the contents of the CSV file!</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/volkamerlab/teachopencadd/pull/84/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='13'/>

Comment (schallerdavid) [inline teachopencadd/talktorials/T011_query_online_api_webservices/talktorial.ipynb:4]: <p>missing word: SARS-CoV-2 is one <strong>of</strong> the most popular</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/volkamerlab/teachopencadd/pull/84/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='21'/>

Comment (schallerdavid) [inline teachopencadd/talktorials/T011_query_online_api_webservices/talktorial.ipynb:4]: <p>typo: datab<strong>a</strong>se</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/volkamerlab/teachopencadd/pull/84/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='38'/>

Comment (schallerdavid) [inline teachopencadd/talktorials/T011_query_online_api_webservices/talktorial.ipynb:4]: <p>The last sentence sounds like some more content will come after the discussion. But there is only the discussion left.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/volkamerlab/teachopencadd/pull/84/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='67'/>

**Comment by schallerdavid** [comment]
Awesome work @jaimergp , the talktorial reads very well! I only found few typos.

**Comment by schallerdavid** [comment]
And the description in the README.md is missing.

**Comment by jaimergp** [comment]
Awesome @schallerdavid! Thanks for the review!

**Comment by jaimergp** [comment]
I'll merge for now and will self-review again when `t011-base` is ready.

**Pull request merged**